fiasco/ions.py

```diff
@@ -61,6 +60,7 @@ def __init__(self, ion_name, temperature: u.K,
         self._dset_names['ioneq_filename'] = kwargs.get('ioneq_filename', 'chianti')
         self._dset_names['ip_filename'] = kwargs.get('ip_filename', 'chianti')
         self.abundance = abundance
+        self.gaunt_factor = GauntFactor()
```

Comment:
```suggestion
        self.gaunt_factor = GauntFactor(hdf5_dbase_root=self.hdf5_dbase_root)
```
You need to pass the same path to the database to the gaunt factor object as is used by the ion object. The CI is failing because it's using the default option which is to look in $HOME/.fiasco but that doesn't exist and it hangs when it asks if you want to build it. The CI runner will use a temporary database built for every test run that is passed through to all of the ion object fixtures. It passes locally for you because you have a local version of the database in ~/.fiasco.

Reply:
Aha!  Thanks.